lib/fs_utils/generate.js

```diff
@@ -188,6 +188,11 @@ const optimize = (data, map, path, optimizers, sourceFiles) => {
   return promiseReduce(optimizers, runOptimizer, initial);
 };
 
+const getInlineSourceMapUrl = map => {
+  const base64String = new Buffer(JSON.stringify(map)).toString('base64');
```

Comment:
`Buffer.from` **for strings** was added in `5.10`, this code will cause `TypeError` in Node 4, because `Buffer.from` will expect `TypedArray`.

Lets leave `new Buffer`, I will move `base64` complexity to helper later.